public/lessonBuilder/lessonItemsController.js

```diff
@@ -103,14 +103,21 @@ angular.module('newApp')
         modules.get($routeParams.id, $routeParams.id2)
         .success(function(module){
             $scope.moduleName = module.name;
+            $scope.module = module;
```

Comment:
This line makes the one above obsolete.  In the view, `{{ moduleName }}` can be changed to `{{ module.name }}`.